python/setup.py

```diff
@@ -72,7 +87,7 @@
 optional_ray_files += ray_project_files
 optional_ray_files += ray_dashboard_files
 
-if "RAY_USE_NEW_GCS" in os.environ and os.environ["RAY_USE_NEW_GCS"] == "on":
+if os.getenv("RAY_USE_NEW_GCS") == "on":
```

Comment:
This is now making the assumption that `RAY_USE_NEW_GCS` is set as an environment variable - is that safe? Why did we change it?

Reply:
Actually `os.getenv` is like `os.environ.get`, it returns `None` if it doesn't exist.

Reply:
Oh, gotcha! Didn't catch that change.